### PR TITLE
Rewrite the Triggers tutorial for clarity and flow

### PR DESCRIPTION
**Pull request by sergetron**
# Changes

Rewrites the Triggers tutorial for clarity and flow.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -1,198 +1,203 @@
...
-### [Install the Pipeline](./pipeline.yaml)
+## Overview
 
-Now we have to install the Pipeline we plan to use and also our Triggers
-resources.
+In this tutorial, you will:
 
-`kubectl -n getting-started apply -f ./docs/getting-started/pipeline.yaml`
+1. Set up a [`Pipeline`](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md) that builds a Docker image using
+   [kaniko](https://github.com/GoogleContainerTools/kaniko) and deploys it locally using the `kubectl image` command. The
```
Comment (savitaashture) [inline docs/getting-started/README.md:11]: `kubectl image` :thinking: 

I think it should be `kubectl apply` or `kubectl create`

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -1,198 +1,203 @@
...
 
-Our Pipeline will do a few things.
+2. Set up an [`EventListener`](https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md) that accepts and processes GitHub push events.
 
-- Retrieve the source code
-- Build and push the source code into a Docker image
-- Push the image to the specified repository
-- Run the image locally
+3. Set up a [`TriggerTemplate`](https://github.com/tektoncd/triggers/blob/main/docs/triggertemplates.md) that instantiates a
+   [`PipelineResource`](https://github.com/tektoncd/pipeline/blob/main/docs/resources.md) and executes a [`PipelineRun`](https://github.com/tektoncd/pipeline/blob/main/docs/pipelinerun.md)
```
Comment (savitaashture) [inline docs/getting-started/README.md:21]:
```suggestion
   [`PipelineResource`](https://github.com/tektoncd/pipeline/blob/main/docs/resources.md) and executes a [`PipelineRun`](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md)/[`TaskRun`](https://github.com/tektoncd/pipeline/blob/main/docs/taskruns.md)
```

File: docs/getting-started/README.md [hunk 3]
```diff
@@ -1,198 +1,203 @@
...
-- A `TriggerTemplate` to create templated PipelineResource and PipelineRun
-  resources per event received by the `EventListener`.
-  - First, **update** the `triggers.yaml` file to reflect the Docker repository
-    you wish to push the image blob to.
-  - You will need to replace the `DOCKERREPO-REPLACEME` string everywhere it is
-    needed.
-  - Once you have updated the triggers file, you can apply it!
-    - `kubectl -n getting-started apply -f ./docs/getting-started/triggers.yaml`
-  - If that succeeded, your cluster is ready to start handling Events.
+Now that you have we have your Kubernetes cluster up and running, you must
```
Comment (savitaashture) [inline docs/getting-started/README.md:43]:
```suggestion
Now that you have your Kubernetes cluster up and running, you must
```

File: docs/getting-started/README.md (hunk 1 above)
Comment (sergetron) [reply docs/getting-started/README.md:11]: I copied this from here: https://github.com/tektoncd/triggers/blob/main/docs/getting-started/README.md#what-does-it-do.  It says "The Pipeline will build a Docker image with kaniko and deploy it locally via kubectl image." What should we say here instead?

File: docs/getting-started/README.md (hunk 2 above)
Comment (sergetron) [reply docs/getting-started/README.md]: Fixed.

File: docs/getting-started/README.md (hunk 3 above)
Comment (sergetron) [reply docs/getting-started/README.md]: Fixed.

File: docs/getting-started/README.md (hunk 1 above)
Comment (sergetron) [reply docs/getting-started/README.md:11]: I'm all for keeping it super-simple unless there's a very specific reason we must point out it's a "kubectl Docker image." Changed it to "...and deploys it locally to your Kubernetes cluster." Let me know if that works.

File: docs/getting-started/README.md [hunk 4]
```diff
@@ -1,198 +1,202 @@
...
-The Pipeline will build a Docker image with
-[kaniko](https://github.com/GoogleContainerTools/kaniko) and deploy it locally via kubectl
-image.
+Before you begin, you must satisfy the following prerequisites:
 
-### [Install the TriggerTemplate, TriggerBinding and EventListener](./triggers.yaml)
+- [Set up a Kubernetes cluster](https://kubernetes.io/docs/setup/) that:
+  - You can publicly access over the Internet,
+  - Allows you to [create ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+  Major cloud providers offer compatible Kubernetes clusters; however, Tekton Triggers does not directly support
```
[the remark this answers is not in the document]
Comment (sergetron) [reply docs/getting-started/README.md:33]: Removed the ingress creation requirement and surrounding text.

File: docs/getting-started/README.md [hunk 5]
```diff
@@ -1,198 +1,199 @@
...
-    needed.
-  - Once you have updated the triggers file, you can apply it!
-    - `kubectl -n getting-started apply -f ./docs/getting-started/triggers.yaml`
-  - If that succeeded, your cluster is ready to start handling Events.
+Now that you have your Kubernetes cluster up and running, you must set up your namespace and RBAC.
+You will keep all of the artifacts for this tutorial within this namespace. This way, you can easily
+start over by deleting and recreating this namespace if necessary. 
 
-## Add Ingress and GitHub-Webhook Tasks
+**Note:** Record your `ingress` sub-dowmain or the external IP address of your
```
Comment (mpetason) [inline docs/getting-started/README.md:44]:
```suggestion
**Note:** Record your `ingress` sub-domain or the external IP address of your
```

**Review by mpetason — COMMENTED** [review]
spelling fix

File: docs/getting-started/README.md (hunk 5 above)
Comment (sergetron) [reply docs/getting-started/README.md]: Fixed.

File: docs/getting-started/README.md [hunk 6]
```diff
@@ -1,198 +1,199 @@
...
 
-### Run the Ingress Task
+Your Tekton stack is now configured to detect and respond to GitHub events.
 
-When you are ready, run the ingress Task.
+## Create and execute the ingress and webhook `Tasks`
 
-`kubectl -n getting-started apply -f docs/getting-started/ingress-run.yaml`
+Now, you must create and execute the following `Tasks`:
+- Ingress `Task` - allows your Tekton stack to detect your specific Github webhook.
```
[the remark this answers is not in the document]
Comment (sergetron) [reply docs/getting-started/README.md]: Got it! Reworded.

File: docs/getting-started/README.md [hunk 7]
```diff
@@ -1,198 +1,199 @@
...
+- Webhook `Task` - creates the Github webhook.
```
[the remark this answers is not in the document]
Comment (sergetron) [reply docs/getting-started/README.md]: Reworded.

**Comment by sergetron** [comment]
Thank you for the thorough reviews Savita and Dibyo! Can we have LGTM?

**Review by dibyom — APPROVED** [review]
/lgtm

**Pull request merged**